Britannia/events/BTWK_frontier_events.txt

```diff
@@ -133,140 +266,318 @@ BTWK_frontier_event.0101 = {
 }
 
 # HADRIAN'S WALL # 
+# Ruin Hadrian's Wall
 BTWK_frontier_event.0201 = {
 	type = character_event
 	title = BTWK_frontier_event.0201
 	desc = BTWK_frontier_event.0201_desc
-	theme = realm
+	theme = stewardship_domain_focus
+	override_background = {
+		reference = courtyard
+	}
+
+	left_portrait = {
+		character = root 
+		animation = personality_rational
+	}	
 
 	option = {
-		# Restore Hadrian's Wall
 		name = BTWK_frontier_event.0201.a
-		trigger = {
-			OR = {
-				is_imperial_culture_trigger = yes
-				has_trait = romanitas_trait
-			}
+		custom_tooltip = BTWK_frontier_event.0201.a_tt
+		add_character_modifier = {
+			modifier = destroyed_hadrians_wall_character_modifier
 		}
 
 		hidden_effect = {
+			save_scope_as = destroyed_hadrians_wall_char
 			every_province = {
 				limit = {
-					geographical_region = custom_hadrians_wall
+					geographical_region = custom_hadrians_wall_provs
+				}
+				if = {
+					limit = {
+						this = {
+							has_building = hadrians_wall_02
+						}
+					}
+					remove_building = hadrians_wall_02
+					add_special_building_slot = hadrians_wall_01
+					add_special_building = hadrians_wall_01
+				}
+				else_if = {
+					limit = {
+						this = {
+							has_building = hadrians_wall_03
+						}
+					}
+					remove_building = hadrians_wall_03
+					add_special_building_slot = hadrians_wall_01
+					add_special_building = hadrians_wall_01
+				}
+				every_county = {
+					limit = {
+							geographical_region = custom_hadrians_wall_provs
+						}
+					}				
+					add_county_modifier = {
+						modifier = destroyed_hadrians_wall_county_modifier
+						years = 20
+					}
+				}
+
+			every_ruler = {
+				limit = {
+					is_landed = yes
+					capital_province = {
+						OR = {
```

Comment:
world_britannia as above?

Reply:
Same question again as above - don't think we have a world_britannia. I can always add it if you like though?